### PR TITLE
Add meeting notes from contribution discussion

### PR DESCRIPTION
**Pull request by MaxWofford**

### CODE REVIEW

File: meetings/15-09-30_contribution_workflow_discussion.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# 09-30.15 Contribution Workflow Discussion
```
Comment (zachlatta) [inline meetings/15-09-30_contribution_workflow_discussion.md]: Can you make that a consistent date format?

File: meetings/15-09-30_contribution_workflow_discussion.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+_Attendees_:
+
+- Dave Fontenot
+- Jonathan Leung
+- Max Wofford
+- Zach Latta
+
```
Comment (zachlatta) [inline meetings/15-09-30_contribution_workflow_discussion.md]: Can you also add a note in this top section that we revised this meeting and link to the new meeting's notes?

**Comment by MaxWofford** [comment]
@zachlatta Comments addressed.

**Comment by MaxWofford** [comment]
@zachlatta I think the tests broke. Could  I get a  second set of eyes on this?

File: meetings/15-09-30_contribution_workflow_discussion.md (hunk 2 above)
Comment (zachlatta) [inline meetings/15-09-30_contribution_workflow_discussion.md]: Ping on this?

Comment (MaxWofford) [inline meetings/15-09-30_contribution_workflow_discussion.md]: This is on the following line.

**Comment by MaxWofford** [comment]
@zachlatta ping

**Comment by zachlatta** [comment]
Sorry about the delay. Merged!

**Pull request merged**